.github/workflows/node-build-deploy.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             ${{ runner.os }}-
       - name: Install deps
         run: |
-          npm install
+          npm install --legacy-peer-deps
```

Comment:
Er dette greit?
npm-versjonen som brukes med node18 er strengere på dependency resolution. Mer info om dette her: https://stackoverflow.com/questions/66239691/what-does-npm-install-legacy-peer-deps-do-exactly-when-is-it-recommended-wh
Hvis jeg fjerner alle avhenigheter til gamle designsystemet så går det fint å kjøre `npm install` uten `--legacy-peer-deps`, så satser på at vi kan fjerne denne når vi en vakker dag har migrert til nye Aksel DS.

Reply:
En vakker dag! 
<img src="https://media0.giphy.com/media/2BU7V1H84fBhK3Qde9/giphy.gif"/>